index.html

```diff
@@ -259,11 +259,12 @@
           </p>
           <ul>
             <li>
-              <code>http://fedora.info/definitions/repository#EmbedResources</code>:
-              Requires a server to include representations of any contained resources in the response.
+              <code>http://www.w3.org/ns/oa#PreferContainedDescriptions</code>:
```

Comment:
I like the idea of reusing existing vocabularies, but am wary of the slight disconnect between Fedora's notion of embedding resources and Open Annotation's notion of descriptions of contained Annotations:
http://w3c.github.io/web-annotation/vocab/wd/#prefercontaineddescriptions

...particularly since the definition of `PreferContainedDescriptions` specifically calls out: `...client prefers to receive full descriptions of the Annotations from a container...`